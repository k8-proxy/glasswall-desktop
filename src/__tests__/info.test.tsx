import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import Info from '../ui/components/Info';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should test Header component', () => {
    const wrapper = shallow(<Info />);
    expect(wrapper).toMatchSnapshot();
});