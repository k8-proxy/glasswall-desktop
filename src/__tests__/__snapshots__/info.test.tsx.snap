// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Header component 1`] = `
<WithStyles(ForwardRef(Card))
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(Grid))
      container={true}
      justify="space-between"
      spacing={3}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <WithStyles(ForwardRef(Typography))
          color="textSecondary"
          gutterBottom={true}
          variant="h6"
        >
          BUDGET
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          color="textPrimary"
          variant="h3"
        >
          $24,000
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <WithStyles(ForwardRef(Avatar))
          className="makeStyles-avatar-2"
        >
          <Memo(ForwardRef(MoneyIcon)) />
        </WithStyles(ForwardRef(Avatar))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <Styled(MuiBox)
      alignItems="center"
      display="flex"
      mt={2}
    >
      <Memo(ForwardRef(ArrowDownwardIcon))
        className="makeStyles-differenceIcon-3"
      />
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-differenceValue-4"
        variant="body2"
      >
        12%
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        color="textSecondary"
        variant="caption"
      >
        Since last month
      </WithStyles(ForwardRef(Typography))>
    </Styled(MuiBox)>
  </WithStyles(ForwardRef(CardContent))>
</WithStyles(ForwardRef(Card))>
`;
